ios/FluentUI/Core/Theme/Tokens/FontInfo.swift

```diff
@@ -76,22 +76,44 @@ public extension Font {
 
 extension UIFont {
     @objc public static func fluent(_ fontInfo: FontInfo, shouldScale: Bool = true) -> UIFont {
-        let unscaledFont: UIFont
+        return fluent(fontInfo, shouldScale: shouldScale, contentSizeCategory: nil)
+    }
+
+    static func fluent(_ fontInfo: FontInfo, shouldScale: Bool = true, contentSizeCategory: UIContentSizeCategory?) -> UIFont {
```

Comment:
Could you add an example of using this somewhere? Right now it's just always nil, so I'm not sure why we've added it.

Reply:
See #1646, where I originally made this change.

Reply:
I think it would still be useful to have an example in this PR, possibly in our unit test library if real-world usage is still incoming?